### ISSUE TITLE
non-www does not resolve DNS

### ISSUE THREAD

**Issue by miki725** [crashappsec/www-qa-issues#41]
```bash
# dig crashoverride.com @1.1.1.1

; <<>> DiG 9.11.3-1ubuntu1.18-Ubuntu <<>> crashoverride.com @1.1.1.1
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 60673
;; flags: qr rd ra ad; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;crashoverride.com.             IN      A

;; AUTHORITY SECTION:
crashoverride.com.      300     IN      SOA     ns-cloud-e1.googledomains.com. cloud-dns-hostmaster.google.com. 15 21600 3600 259200 300

;; Query time: 107 msec
;; SERVER: 1.1.1.1#53(1.1.1.1)
;; WHEN: Tue Sep 27 14:49:50 UTC 2022
;; MSG SIZE  rcvd: 136
```

as a result not all browsers auto-redirect to `www.` variant and you get errors:

### firefox

![image](https://user-images.githubusercontent.com/932940/192560708-5af7b776-5e0d-4b40-b2be-314d68ecf659.png)

### brave

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/932940/192560790-fd7020dd-8af7-44ab-b655-bf6bca2e7b78.png">

### chrome

<img width="1069" alt="image" src="https://user-images.githubusercontent.com/932940/192560851-e3032754-252d-490e-9631-a6d6f51320f3.png">

### safari

interesting on safari it works out of the box if you simply type `crashoverride.com` as it auto-adds `www.`:

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/932940/192562260-c83a6fc9-9ec9-4026-a871-68b2da27dfd2.png">

but if you explicitly try `https://crashoverride.com` error:

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/932940/192562213-7a6d15c5-8a79-4e97-9886-f96ada6d4384.png">

**Comment by curphey** [comment]
John, can you set a DNS redirect to take care of this ?

**Comment by viega** [comment]
No, you cannot add a CNAME record for the host domain.  I can add an A record, but it means the Web server needs to know that it is also crashoverride.com and there has to be a cert in place for it (assuming the one in place is not a wildcard, which I hope it isn't).

**Comment by miki725** [comment]
I think this might be it https://support.google.com/domains/answer/4522141?hl=en

**Comment by viega** [comment]
I've done that for other domains such as crashøverride.com but when you do it, I believe it uses its own proxy to rewrite, and does it across www and the non-www.  Hubspot required the www to be configured a particular way, so doing this I'd expect to break Hubspot, at least eventually.  If I point it directly at hubspot's underlying pool, it'd still need to know to be authoritative.  But I can try it and 1) hope it doesn't clobber our existing www entry, and 2) if it does, see if Hubspot throws up on it... @curphey, your call here.

**Comment by miki725** [comment]
guess this is where cloudflare page rules are pretty useful. you can have a rule per page url and if you proxy all requests through cloudflare, they just do all the conditionals on the edge servers. not sure if google domains has something similar

but a silly question. why www to begin with? why not have the root domain be the main site?

**Comment by viega** [comment]
IDK, I am actioning what Mark's requesting, based on Hubspot's recommendations :). But Hubspot does support this, so the site works now.

**Comment by miki725** [comment]
```
V dig crashoverride.com +short
199.60.103.135
199.60.103.35
```

🎉

**Comment by miki725** [comment]
```
➜ dig crashoverride.com

; <<>> DiG 9.18.6 <<>> crashoverride.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 21135
;; flags: qr rd ra ad; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;crashoverride.com.             IN      A

;; AUTHORITY SECTION:
crashoverride.com.      174     IN      SOA     ns-cloud-e1.googledomains.com. cloud-dns-hostmaster.google.com. 17 21600 3600 259200 300

;; Query time: 0 msec
;; SERVER: 10.10.20.1#53(10.10.20.1) (UDP)
;; WHEN: Mon Oct 03 11:29:33 EDT 2022
;; MSG SIZE  rcvd: 136
```

this is happening to me again. did DNS change?

cc @viega
